Makefile

```diff
@@ -183,3 +183,19 @@ clean:
 		cover.out oci-archives bundle bundle.Dockerfile \
 		skupper-*.tgz artifacthub-repo.yml \
 		network-observer-*.tgz  skupper-*-scope.yaml
+
+# Run a specific test
+test:
+	$(MAKE) -C tests/ test TEST="$(TEST)"
+
+# Run all e2e tests
+test-e2e:
+	$(MAKE) -C tests/ e2e-tests
```

Comment:
Actually why don't we bring in only one new target, which could be this one: `test-e2e`.
A single target could do the work of invoking subtests if a variable is provided.

This would make it clearer as we'd have a single entrypoint to run e2e tests.

Reply:
Done under: 4cf39555a81086798f6bb787ceef1354227a4224